terraform-modules/stackdriver/k8s-cluster-monitoring/k8s-cluster-health.tf

```diff
@@ -0,0 +1,59 @@
+/*
+Creates an alert policy which monitors the ratio of
+pods currently running in a cluster to the max number of pods
+a cluster can support.
+It will trigger the alert if current pods running are above 95% max capacity for number of pods 
+*/
+
+locals {
+  cluster_pod_capacity_metric = "metric.type=\"external.googleapis.com/prometheus/kube_node_status_capacity_pods\" resource.type=\"k8s_container\""
+  cluster_pod_running_metric  = "metric.type=\"external.googleapis.com/prometheus/kube_pod_status_phase\" resource.type=\"k8s_container\" metric.label.\"phase\"=\"Running\""
+  # Indicates ratio of number of currently running pods to max number of pods cluster can support
+  cluster_pod_capacity_threshold = 0.95
+}
+
+resource google_monitoring_alert_policy cluster_health_check {
+
+  provider              = google.target
+  display_name          = "k8s-cluster-pod-capacity"
+  project               = var.project
+  combiner              = local.condition_combine_method
+  enabled               = true
+  notification_channels = var.notification_channels
+  user_labels = {
+
+  }
```

Comment:
If we're not setting this field and not exposing a variable to override it we can probably omit it since it's optional?

Reply:
oops forgot to take these out, thanks!